pages/common/pssh.md

```diff
@@ -0,0 +1,27 @@
+# pssh
+
+> Parallel ssh program.
+
+- Run a command on host1 and host2 to print the date on each server inline:
+
+`pssh -i -H "{{host1}} {{host2}}" date`
+
+- Run a command and save the output to separate files:
+
+`pssh -H {{host1}} -H {{host2}} -o {{path/to/output_dir}} date`
+
+- Run a command on each host specified in a new-line separated host file:
+
+`pssh -i -h {{path/to/hosts_file}} date`
+
+- Run a commond as root, prompting for root's password:
+
+`pssh -i -h {{path/to/hosts_file}} -A -l root date`
```

Comment:
Ok, can we change the description to -
`Run a command as root; asks for root password` ?

Just want to match the option `-A` with a word in the description.

Also, typo in `commond`.